modules/core/04-channel/keeper/keeper.go

```diff
@@ -532,13 +532,25 @@ func (k Keeper) SetCounterpartyLastPacketSequence(ctx sdk.Context, portID, chann
 	store.Set(host.ChannelCounterpartyLastPacketSequenceKey(portID, channelID), bz)
 }
 
+// deleteCounterpartyLastPacketSequence deletes the counterparty last packet sequence from the store.
+func (k Keeper) deleteCounterpartyLastPacketSequence(ctx sdk.Context, portID, channelID string) {
```

Comment:
ping: added these separately in https://github.com/cosmos/ibc-go/pull/3780 so as to re-use them in any pr's required.